### PR TITLE
#532 - Add code coverage support - Jacoco

### LINKED ISSUE #532: Add code coverage support - JaCoCo
## Background
Add code coverage support to be able measure code quality.

## Feature
New ability to measure current code coverage as one of QA metrics.

### PR DESCRIPTION
**Pull request by miroslavpojer**
- added new plugin dependency
- extended readme.md file with how to get code coverage report
- extended build logic to add JaCoCo non-default settings

Closes #532

### CODE REVIEW

**Comment by yruslan** [comment]
After this is merged we can remove `sbt-scoverage` from the project, right?

**Review by yruslan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1423,6 +1432,16 @@ For multisegment variable lengths tests:
...
+
+### How to generate Code coverage report
+```sbt
+sbt jacoco
+```
+Code coverage will be generated on path:
+```
+{local-path}\fixed-width\target\scala-2.XY\jacoco\report\html
+```
+
```
Comment (Zejnilovic) [inline README.md:1444]: This might be extra

Comment (miroslavpojer) [reply README.md:1444]: Yes, I was in doubt if it even needed to place in readme when I saw that file. Remove it?

**Comment by miroslavpojer** [comment]
> After this is merged we can remove `sbt-scoverage` from the project, right?

It depends. You are used to use it. So now we can compare additional value of both.

**Pull request merged**